src/zend/globals.rs

```diff
@@ -51,7 +51,11 @@ impl ExecutorGlobals {
         // return an invalid pointer.
         let globals = unsafe { ext_php_rs_executor_globals().as_ref() }
             .expect("Static executor globals were invalid");
-        let guard = GLOBALS_LOCK.read();
+        #[cfg(not(php_zts))]
+        let guard = lock::GLOBALS_LOCK.read_arc();
+        #[cfg(php_zts)]
+        let guard = lock::GLOBALS_LOCK.with(|l| l.read_arc());
```

Comment:
Use `cfg-if` for those. Makes the `if`/`else` nature of these more obvious.

Reply:
done